apps/builder/app/builder/shared/assets/image-formats.ts

```diff
@@ -0,0 +1,49 @@
+import { nanoid } from "nanoid";
+
+const extensionToMime = new Map([
+  [".gif", "image/gif"],
+  [".ico", "image/x-icon"],
+  [".jpeg", "image/jpeg"],
+  [".jpg", "image/jpeg"],
+  [".png", "image/png"],
+  [".svg", "image/svg+xml"],
+  [".webp", "image/webp"],
+] as const);
```

Comment:
looks like we have something similar in asset manager package

Reply:
Yep but there ton of types we don't need in UI is supported